app/services/manifest_builder_v3/record_property_builder.rb

```diff
@@ -7,17 +7,7 @@ class RecordPropertyBuilder < IIIFManifest::V3::ManifestBuilder::RecordPropertyB
     def apply(manifest)
       manifest = super
       manifest.viewing_direction = viewing_direction if viewing_direction.present? && manifest.respond_to?(:viewing_direction=)
-      manifest.behavior = ["auto-advance"]
-      if record.resource.downloadable == ["none"]
-        manifest["service"] ||= []
-        manifest["service"] << {
-          "@context" => "http://universalviewer.io/context.json",
-          "profile" => "http://universalviewer.io/ui-extensions-profile",
-          "disableUI" => ["mediaDownload"]
```

Comment:
So if we remove this, does the download pane still go away for things where download is disabled?

Reply:
It does not go away, but says there are no downloads. If you want to make this happen going forward (once we upgrade UV), I think we'll have to somehow pass the [uv_config_downloads_disabled](https://github.com/pulibrary/figgy/blob/main/public/uv/uv_config_downloads_disabled.json) config that we have. Though I don't see anyplace where we are using it currently. I

Reply:
That's surprising, I wonder if it's the config disabling the downloads and we're fine here? I would have expected it to still show that like, "download a jpg" selection that uses the IIIF image API